themes/bootstrap3/templates/Helpers/identifierLinks.phtml

```diff
@@ -0,0 +1,14 @@
+<?php
+    $attribs = ['class' => 'identifierLink', 'data-instance' => $instance ?? 0];
+    if (!empty($doi)) {
+        $attribs['data-doi'] = $doi;
+    }
+    if (!empty($isbn)) {
+        $attribs['data-isbn'] = $isbn;
+    }
+    if (!empty($issn)) {
+        $attribs['data-issn'] = $issn;
+    }
+?>
+<?=$this->inlineScript(\Laminas\View\Helper\HeadScript::FILE, 'identifierLinks.js', 'SET');?>
```

Comment:
Maybe I'm missing something here, but the script might not be available, if for no record in a result list on the first page the identifier linker is active but for records on other pages later loaded via JS.

Reply:
Since this is loaded with an inline script, I believe the script should dynamically load even if the records get pulled in via AJAX. It would probably be more efficient to load this once in the layout, but I think it should work without (though if you can find a way to break it, I'd be interested to hear it).